gsa/src/web/components/menu/menu.js

```diff
@@ -114,13 +116,24 @@ const MenuList = styled.ul`
   }
 `;
 
+const checkChild = child => {
+  if (child.type === MenuSection && isArray(child.props.children)) {
```

Comment:
I am not 100 percent sure if this will work. React has a special API to access children because they aren't arrays https://reactjs.org/docs/react-api.html#reactchildren